flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/InputChannelTestUtils.java

```diff
@@ -150,7 +154,7 @@ public static RemoteInputChannel createRemoteInputChannel(
 			connectionManager,
 			0,
 			0,
-			UnregisteredMetricGroups.createUnregisteredTaskMetricGroup().getIOMetricGroup());
+			new InputChannelMetrics(UnregisteredMetricGroups.createUnregisteredTaskMetricGroup().getIOMetricGroup()));
```

Comment:
`InputChannelTestUtils.newUnregisteredInputChannelMetrics()`?

Reply:
I guess we define a new method in `InputChannelTestUtils` for creating this `InputChannelMetrics` in order to reuse in multiple places?

Reply:
I suggest so